### DISCUSSION TITLE
Page to view promoted content at once

### DISCUSSION THREAD

**Discussion by kavinda1995** [dailydotdev/daily#773] [Feature Requests]
daily.dev's promoted contents are ads - of course. But most of the times, they introduce us to new services and tools that we are not even knew before - So it's a good thing ❤️. 

Today, I saw an promoted content like that, which was really interesting. But unfortunately, before I click on that I got my tab closed. I opened a new tab (where I have settled daily.dev as default new tab page) to find it. But it wasn't there. So I kept on refreshing the tab around ~20 times to get that content again loaded up in promoted content - and finally got it. 

This leads me to think, wouldn't it be good if we have some new page like `Read history` or `Boookmarks` - but only showing the promoted contents. The only reason for this is that promoted content are placed there randomly, unlike others.

WDYT?

**Comment by idoshamun** [comment]
Hi @kavinda1995,

That's interesting feedback! I love your point of view and understand the new to discover new tools and services.
Let me share it internally and we'll think how to address that
